stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -170,6 +170,7 @@ data class PaymentMethodCreateParams internal constructor(
         Alipay("alipay"),
         GrabPay("grabpay"),
         PayPal("paypal"),
+        AfterPay("afterpay_clearpay"),
```

Comment:
`Afterpay`?

Reply:
idk why I thought it was stylized as AfterPay